book/B-embedding-git/sections/libgit2.asc

```diff
@@ -3,7 +3,7 @@
 (((libgit2)))((("C")))
 Another option at your disposal is to use Libgit2.
 Libgit2 is a dependency-free implementation of Git, with a focus on having a nice API for use within other programs.
-You can find it at http://libgit2.github.com[].
+You can find it at https://github.com/libgit2/libgit2[].
```

Comment:
This should probably link to https://libgit2.org/.

Reply:
I've changed the link. :+1: